test/tests/pbs_smoketest.py

```diff
@@ -1371,17 +1378,27 @@ def test_resource_delete(self):
                                 id=self.resc_name, logerr=False)
         except (PbsManagerError, PbsStatusError):
             pass
-        for k in self.objs:
-            if k not in self.obj_map:
-                self.logger.error('can not map object ' + k)
-                continue
-            v = self.obj_map[k]
-            for t in self.resc_types:
-                for f in self.resc_flags:
-                    for c in self.resc_flags_ctl:
-                        self.delete_resource_helper(
-                            self.resc_name, t, f, c, k, v)
-                        self.logger.info("")
+        count = 0
+        starttime = int(time.time())
```

Comment:
Can you please explain why this change is needed? was this test failing for you?

Reply:
Yes this test was failing.  When we would run this in debian9 with python 3.5 and if there was type hinting code inside the python, it would take much longer to run.  There is some kind of bug in python 3.5 that causes this.  I think the version was specifically 3.5.3.  This test seems to be the most sensitive to time and I added a timer so that when it fails in the ptl, we can compare it to the other tests and also still get the error.  Brian might be able to explain it better as he dug into it.

Reply:
From commit 9993b91f2f99598ef4b2b81f9f4fe07bbd12eea8 (with a bit of clean up):

>The Python typing module seems to be unstable in Python 3.5.3, which is what Debian 9 uses.  With the type specifications in place, Py_Finalize() would take increasing longer on each Python interpreter restart, eventually taking more than two seconds to complete and causing the SmokeTest.test_resource_delete to timeout.  Removing the type specifications results in the test successfully completion in the expected amount of time.

BTW, the issue disappeared when we tested with Python 3.5.9.  The change log for Python 3.5 noted updates to the typing module between 3.5.3 and 3.5.9, so I'm guessing they fixed a bug in 3.5.3 that Debian didn't backport.

Reply:
In that case, can we remove this change? I am not sure we should add this change for something that one Linux distribution failed to backport.

Reply:
I think we should leave it that way if there is something else that slows this down, we can get timings on what happened.  All it's doing is showing how long it took to iterate through all the tests.  This also may be associated with random failures in here: https://pbspro.atlassian.net/browse/PP-359

Reply:
ok, I agree. Let us leave it as is.
Thanks!